packages/libs/client-utils/src/built-in/describe-module.ts

```diff
@@ -1,31 +1,42 @@
 import { execution } from '@kadena/client/fp';
 import { pipe } from 'ramda';
+import { z } from 'zod';
 import { dirtyReadClient } from '../core/client-helpers';
 import type { IClientConfig } from '../core/utils/helpers';
 
 /**
  * @alpha
  */
-export interface IDescribeModuleOutput {
```

Comment:
I think its better to keep the interface, then the type is more clear for end user.
What do you think?

Reply:
Yeah, ofcourse, no issue at all. Will refactor it to be an `interface` again 👍